app/assets/stylesheets/components/video.css

```diff
@@ -17,4 +17,9 @@
   .v-bigPlay {
     opacity: 0.5 !important;
   }
+  
+  .v-openInYouTube.v-controlButton svg {
+    width: 26px;
```

Comment:
I think we should try to avoid px base sizes. For now if I zoom in with my browser the other buttons are getting bigger t. This new button should have the same behaviour

Reply:
Hmm, I do agree with avoiding base px sizes. But I use Edge and Firefox browsers and on both the svg dimensions stay consistent. I did not see any changes on the icon so I left it as it. I'll update it to not use px size.